### PR TITLE
Pass more wiki request info to the AI for consideration

### PR DESCRIPTION
**Pull request by AgentIsai**

### CODE REVIEW

**Comment by Universal-Omega** [comment]
This should be hooks. This undoes all the work I've done to CreateWiki by hardcoding values into it again. For now this should be reverted and hooks done for data that doesn't have anything to do with CreateWiki itself.

File: includes/Services/WikiRequestManager.php [hunk 1]
```diff
@@ -975,7 +985,14 @@ private function tryAutoCreate( string $reason ): void {
 	private function evaluateWithOpenAI(
 		string $sitename,
 		string $subdomain,
-		string $reason
+		string $reason,
+		string $username,
+		string $language,
+		bool $bio,
+		bool $private,
```
Comment (Universal-Omega) [inline includes/Services/WikiRequestManager.php:992]: These orders are all out of whack as well.

**Comment by AgentIsai** [comment]
> This should be hooks. This undoes all the work I've done to CreateWiki by hardcoding values into it again. For now this should be reverted and hooks done for data that doesn't have anything to do with CreateWiki itself. At the very least for nsfwtext

If you can figure it out, that'd be great. You know well this code will be scrapped all together come June hence why it's been included.

**Comment by Universal-Omega** [comment]
> > This should be hooks. This undoes all the work I've done to CreateWiki by hardcoding values into it again. For now this should be reverted and hooks done for data that doesn't have anything to do with CreateWiki itself. At the very least for nsfwtext
> 
> If you can figure it out, that'd be great. You know well this code will be scrapped all together come June hence why it's been included.

I suppose that is fair. Sorry if I overreacted lol. Just after doing all the work on CW I kinda panicked when seeing those values hardcoded. I may do something later, for now I suppose it is fine.

File: includes/Jobs/RequestWikiRemoteAIJob.php [hunk 2]
```diff
@@ -249,10 +274,25 @@ private function handleLiveRun( string $outcome, string $comment ): bool {
...
-			$sanitizedReason = "Wiki name: $sitename. Subdomain: $subdomain. Wiki request reason: " .
+			$isBio = $bio ? "Yes" : "No";
+			$isPrivate = $private ? "Yes" : "No";
+			$isNsfw = $nsfw ? "Yes" . $nsfwtext : "No";
+			$nsfwReasonText = $nsfw ? "What type of NSFW content will it feature? '$nsfwtext'. " : "";
+
+			$sanitizedReason = "Wiki name: '$sitename'. Subdomain: '$subdomain'. Requester: '$username'. " .
+			"Language: '$language'. Focuses on real people/groups? '$isBio'. Private wiki? '$isPrivate'. " .
+			"Category: '$category'. Contains content that is not safe for work? '$isNsfw'. " .
+			$nsfwReasonText . "Wiki request reason: " .
```
Comment (Universal-Omega) [inline includes/Jobs/RequestWikiRemoteAIJob.php:295]: Oh though this seeks like a real problem that will significantly confuse the AI?

**Pull request merged**